auth-client/src/main/java/gov/usgs/cida/auth/client/AuthClient.java

```
@@ -83,6 +86,15 @@ public AuthToken getNewToken(String username, String password) {
 					ex.getResponse().getStatus() == Response.Status.UNAUTHORIZED.getStatusCode()) {
 				throw new NotAuthorizedException(ex.getMessage());
```

Any exception that you know is likely caused by a not-authorized status should just throw this runtime exception.